cmd/buildah/rm.go

```diff
@@ -31,27 +38,45 @@ func rmCmd(c *cli.Context) error {
 	}
 
 	var e error
-	for _, name := range args {
-		builder, err := openBuilder(store, name)
-		if e == nil {
-			e = err
-		}
+	if c.Bool("all") {
+		builders, err := openBuilders(store)
```

Comment:
This returns the set of containers which buildah is using, but would not include those which it is not (for example, any created by cri-o).  Is this the intention?

Reply:
I thought it was just to do Buildah containers.  @rhatdan?

Reply:
That is I meant it to remove the containers shown with "buildah containers", holler if this doesn't hit that mark, seems to in my testing, but I didn't try adding a container via Podman in my tests.

Reply:
Yes we need to be careful with this going forward,  We have to figure out when we merge cri-o/podman/buildah together, what is the expected output of this.

If buildah images

List all images, which I think it should eventually.  I would want buildah rmi -a to remove them all whether they are created via buildah or other tools.

Reply:
We set a `*` in the `BUILDER` column in the `buildah containers` output if the container has some buildah-specific data associated with it, and containers which don't (i.e., created by cri-o, and not yet touched by buildah) are not listed unless `buildah containers` was passed the `-a` flag.  The two `-a` flags don't seem to work the same way.

Reply:
Hmm, sold.  I've to do rmi too, I'll test/look into the containers without the '*' tag too.